.gitignore

```diff
@@ -1,10 +1,13 @@
 /.coverage
+/cdk.out/
 /html/
 /htmlcov/
 /.idea
 /.mutmut-cache
 /.mypy_cache
 /mutmut.xml
+/lambda_functions/*/packages/*
```

Comment:
The `*` at the end is redundant; ignoring a directory ignores all its contents as well.